docs/pages/connect-your-client/putty-winscp.mdx

```diff
@@ -1,16 +1,19 @@
 ---
-title: Using PuTTY to connect to SSH nodes via Teleport
-description: This reference shows you how to use PuTTY to connect to SSH nodes through Teleport
+title: Using PuTTY to connect to SSH nodes and WinSCP to transfer files through Teleport
```

Comment:
I think it would make sense to shorten the title, e.g., "Using PuTTY and WinSCP with Teleport".